lib/Raven/Configuration.php

```diff
@@ -695,27 +659,29 @@ public function getErrorTypes()
      */
     public function setErrorTypes($errorTypes)
     {
-        $options = array_merge($this->options, ['error_types' => $errorTypes]);
-
-        $this->options = $this->resolver->resolve($options);
+        $this->mergeAndResolve(['error_types' => $errorTypes]);
     }
 
     /**
      * Configures the options for this processor.
      *
      * @param OptionsResolver $resolver The resolver for the options
+     *
+     * @throws \Symfony\Component\OptionsResolver\Exception\UndefinedOptionsException
+     * @throws \Symfony\Component\OptionsResolver\Exception\AccessException
      */
     private function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
             'send_attempts' => 6,
             'trust_x_forwarded_proto' => false,
             'prefixes' => explode(PATH_SEPARATOR, get_include_path()),
+            'serializer' => new Serializer(),
+            'reprSerializer' => new Serializer(),
             'serialize_all_object' => false,
             'sample_rate' => 1,
             'install_default_breadcrumb_handlers' => true,
             'install_shutdown_handler' => true,
-            'mb_detect_order' => null,
```

Comment:
I've dropped this option since it's something completely and uniquely related to the serializers, so I don't see the need of exposing this setting here.